### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by kale1d0code**
remove as many RUN commands as possible to reduce the amount of layers in image and attempt to get its weight down (855.86 MB Compressed!)

### CODE REVIEW

**Comment by oliverguenther** [comment]
Hi @kale1d0code , thanks for the contribution. @crohr / @machisuji could you take a look at this PR?

File: docker/prod/Dockerfile [hunk 1]
```diff
@@ -70,13 +70,11 @@ RUN \
...
 
 # Copy lock file again as the updated version was overriden by COPY just now
-RUN cp Gemfile.lock.bak Gemfile.lock && rm Gemfile.lock.bak
-
-RUN ./docker/prod/setup/precompile-assets.sh
-RUN ./docker/prod/setup/postinstall-common.sh
-
-RUN cp ./config/database.production.yml config/database.yml
-RUN ln -s $APP_PATH/docker/prod/setup/.irbrc /home/$APP_USER/
+RUN cp Gemfile.lock.bak Gemfile.lock && rm Gemfile.lock.bak \ &&
```
Comment (machisuji) [inline docker/prod/Dockerfile]:
```suggestion
RUN cp Gemfile.lock.bak Gemfile.lock && rm Gemfile.lock.bak && \
```

**Review by machisuji — CHANGES_REQUESTED** [review]
Looks good overall, thanks! 👍 I have just suggested one fix that's necessary to fix the build.

**Comment by crohr** [comment]
Did it bring meaningful improvements? In any case, this looks fine to me.

**Comment by oliverguenther** [comment]
I'll merge it and we can observe the flavour builds for it.

**Pull request merged**